src/state/models/content/post-thread.ts

```diff
@@ -130,7 +130,12 @@ export class PostThreadItemModel {
         parentModel._showChildReplyLine = true
         if (v.parent.parent) {
           parentModel._showParentReplyLine = true
-          parentModel.assignTreeModels(v.parent, higlightedPostUri, true, false)
+          parentModel.assignTreeModels(
+            v.parent,
+            highlightedPostUri,
+            true,
+            false,
+          )
```

Comment:
The linter made me do this.

Speaking of which, I have some [commits to make the linter friendlier](https://github.com/bluesky-social/social-app/compare/15c1b6ee157471807a723161066ba4ce5e12c0b5..2d11d3f3dcaecc7015954b072f04eb9b95cbf322)